.circleci/docker/common/install_conda.sh

```diff
@@ -97,11 +97,12 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # Pin SciPy because of failing distribution tests (see #60347)
   # Pin MyPy version because new errors are likely to appear with each release
   # Pin hypothesis to avoid flakiness: https://github.com/pytorch/pytorch/issues/31136
+  # Pin unittest-xml-reporting to freeze printing test summary logic, related: https://github.com/pytorch/pytorch/issues/69014
```

Comment:
Do we know why this is the version that has the right skip logic? /Is there a range of versions for which printing doesn;t work?

Reply:
Good question - will need thorough testing to answer though. So far I've found a version that definitely works. Will check

Reply:
Should be fine since v2.0.0 (where infoclass attribute was added): https://github.com/xmlrunner/unittest-xml-reporting/commit/606db78d5c33d69e1acec02ae0d5a204cd3368df

Reply:
Could this either be posted in the issue related or in the comment itself?